mesh/mesh.cpp

```diff
@@ -8890,14 +8890,14 @@ bool Mesh::DerefineByError(const Vector &elem_error, double threshold,
 }
 
 
-void Mesh::InitFromNCMesh(const NCMesh &ncmesh)
+void Mesh::InitFromNCMesh(const NCMesh &other_ncmesh)
```

Comment:
I'd prefer `ncmesh_`, which is the convention used elsewhere in constructors (or "constructor-like" functions)

Reply:
Done